Sources/TypeChecker/TypeChecker+Expressions.swift

```diff
@@ -111,6 +111,11 @@ extension TypeChecker {
       // Ensure an event call's arguments match the expected types.
 
       for argument in functionCall.arguments {
+        guard let _ = argument.identifier else {
```

Comment:
Preferred is `argument.identifier != nil` as it's clearer what you're doing. Xcode actually has a warning when you try to do a null check with a conditional binding.